Task/Workflow/TSKWorkflow.h

```diff
@@ -142,13 +144,13 @@ extern NSString *const TSKWorkflowTaskKey;
 @property (nonatomic, strong, readonly) NSNotificationCenter *notificationCenter;
```

Comment:
Hrm. But we allow you to change it after the fact. Probably should just make it readonly, but that might not be an option at this point. Maybe we make it null_resettable?